src/index.ts

```diff
@@ -73,17 +73,17 @@ export class RetryChunkLoadPlugin {
           if(typeof ${RuntimeGlobals.require} !== "undefined") {
             var oldGetScript = ${RuntimeGlobals.getChunkScriptFilename};
             var oldLoadScript = ${RuntimeGlobals.ensureChunk};
-            var queryMap = new Map();
-            var countMap = new Map();
+            var queryMap = {};
+            var countMap = {};
             var getRetryDelay = ${getRetryDelay}
             ${RuntimeGlobals.getChunkScriptFilename} = function(chunkId){
               var result = oldGetScript(chunkId);
-              return result + (queryMap.has(chunkId) ? '?' + queryMap.get(chunkId)  : '');
+              return result + (queryMap[chunkId] !== undefined ? '?' + queryMap[chunkId]  : '');
             };
             ${RuntimeGlobals.ensureChunk} = function(chunkId){
               var result = oldLoadScript(chunkId);
               return result.catch(function(error){
-                var retries = countMap.has(chunkId) ? countMap.get(chunkId) : ${maxRetries};
+                var retries = countMap[chunkId] !== undefined ? countMap[chunkId] : ${maxRetries};
```

Comment:
`Object.hasOwnProperty` is a more direct equivalent of `Map.has` that preserves the exact original behavior

```suggestion
                var retries = countMap.hasOwnProperty(chunkId) ? countMap[chunkId] : ${maxRetries};
```